national-insurance.conf

```diff
@@ -5239,6 +5268,157 @@ class-one {
       }
     }
   }
+
+  2019 {
+    "Up to LEL" {
+      year: "[0,6136)"
+      month: "[0,512)"
+      week: "[0,118)"
+      employee.ABCHJMZ = 0%
+      employer.ABCHJMZ = 0%
+    }
+    "LEL to PT" {
+      year: "[6136,8632)"
+      month: "[512,719)"
+      week: "[118,166)"
+      employee.ABCHJMZ = 0%
+      employer.ABCHJMZ = 0%
+    }
+    "LEL to ST" {
+      year: "[6136,8632)"
+      month: "[512,719)"
+      week: "[118,166)"
+      employee.ABCHJMZ = 0%
+      employer.ABCHJMZ = 0%
+    }
+    "PT to UEL" {
+      year: "[8632,50000)"
+      month: "[719,4167)"
+      week: "[166,962)"
+      employee {
+        B = 5.85%
+        AHM = 12%
+        JZ = 2%
+        C = 0%
+      }
+    }
+    "ST to UEL" {
+      year: "[8632,50000)"
+      month: "[719,4167)"
+      week: "[166,962)"
+      employer {
+        ABCJ = 13.8%
+        HMZ = 0%
+      }
+    }
+    "Above UEL" {
+      year: "[50000,∞)"
+      month: "[4167,∞)"
+      week: "[962,∞)"
+      employee {
+        ABHJMZ = 2%
+        C = 0%
+      }
+      employer.ABCHJMZ = 13.8%
+    }
+    "Above UST" {
+      year: "[50000,∞)"
+      month: "[4167,∞)"
+      week: "[962,∞)"
+
+      employer {
+        MZ = 13.8%
+        ABCHJ = 0%
+      }
+    }
+    "Employees Rebate" {
+      year: "[6136,0)"
+      month: "[512,0)"
+      week: "[118,0)"
+      employee.ABCHJMZ = 0%
+    }
+    "Employers Rebate" {
+      year: "[6136,0)"
+      month: "[512,0)"
+      week: "[118,0)"
+      employer.ABCHJMZ = 0%
+    }
+  }
+  2020 {
+    "Up to LEL" {
+      year: "[0,6420)"
+      month: "[0,520)"
+      week: "[0,120)"
+      employee.ABCHJMZ = 0%
+      employer.ABCHJMZ = 0%
+    }
+    "LEL to PT" {
+      year: "[6420,9500)"
+      month: "[520,792)"
+      week: "[120,183)"
+      employee.ABCHJMZ = 0%
+      employer.ABCHJMZ = 0%
+    }
+    "LEL to ST" {
+      year: "[6420,8788)"
+      month: "[520,732)"
+      week: "[120,169)"
+      employee.ABCHJMZ = 0%
+      employer.ABCHJMZ = 0%
+    }
+    "PT to UEL" {
+      year: "[9500,50000)"
+      month: "[792,4167)"
+      week: "[183,962)"
+      employee {
+        B = 5.85%
+        AHM = 12%
+        JZ = 2%
+        C = 0%
+      }
+    }
+    "ST to UEL" {
+      year: "[8788,50000)"
+      month: "[732,4167)"
+      week: "[169,962)"
```

Comment:
the lower limit should be open - the band starts at £169.01 according to this: https://www.gov.uk/national-insurance-rates-letters

Reply:
and then the upper limit looks like it should be closed